### PR TITLE
section 7.5

### PR DESCRIPTION
**Pull request by supervladimir**

### CODE REVIEW

File: ptx/sec_work.ptx [hunk 1]
```diff
@@ -523,7 +530,14 @@
           <caption>Illustrating a water tank in order to compute the work required to empty it in <xref ref="ex_pump1">Example</xref></caption>
           <!-- START figures/fig_work_pump1.tex -->
           <image xml:id="img_pump1a" width="47%">
-            <description></description>
+            <description>
+              Image of a cylindrical storage tank with a radius of 10ft and a height of 30ft.
```
Comment (sean-fitzpatrick) [inline ptx/sec_work.ptx]: Add a space between the number and the unit. Even better: use the built-in markup for quantities:
`<quantity><mag>10</mag><unit base="foot"/></quantity>`

Comment (sean-fitzpatrick) [reply ptx/sec_work.ptx]: Same comment applies for further descriptions below, but I'll avoid repeating it each time.

File: ptx/sec_work.ptx [hunk 2]
```diff
@@ -700,7 +721,15 @@
           <caption>A graph of the conical water tank in <xref ref="ex_pump2">Example</xref></caption>
           <!-- START figures/fig_work_pump3.tex -->
           <image xml:id="img_pump2" width="47%">
-            <description></description>
+            <description>
+              Image of a conical storage tank with its tip 10 feet below ground and the base at ground level with a radius of 2ft.
```
Comment (sean-fitzpatrick) [inline ptx/sec_work.ptx]: Here you should abbreviate either both instances of 'feet', or neither.

**Review by sean-fitzpatrick — COMMENTED** [review]
Some minor adjustments to how units are handled, and then this is good to go.

**Comment by supervladimir** [comment]
The measurements should be fixed now

**Pull request merged**